### PR TITLE
German Translation work

### PR DESCRIPTION
**Pull request by kayhayen**
@Fire-Cube lets discuss here

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -219,6 +219,24 @@ serious update from my side, maybe ignore that, until I get there.
...
 
+******************
+ How to translate
+******************
+
+.. code:: bash
+   
+   # Generate the .pot files
+   cd intl
+   make gettext
```
Comment (kayhayen) [inline README.rst:230]: the `make` is kind of windows only for most people, problem?

Comment (kayhayen) [reply README.rst:230]: seems we could make it an invoke target, not a lot in the makefile

File: locales/de_DE/LC_MESSAGES/commercial.po [hunk 2]
```diff
@@ -0,0 +1,394 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2022, Kay Hayen and Nuitka Contributors
+# This file is distributed under the same license as the Nuitka the Python
+# Compiler package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
```
Comment (kayhayen) [inline locales/de_DE/LC_MESSAGES/commercial.po:5]: Da sollte wohl @Fire-Cube email, etc stehen

**Comment by Fire-Cube** [comment]
okay

**Review by kayhayen — COMMENTED** [review]
Will merge when I return home and CI resumes main site updating

File: README.rst (hunk 1 above)
Comment (Fire-Cube) [reply README.rst:230]: it's easy to install make on windows

**Pull request merged**